src/UI/inputs/MyInputs.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import cl from '../inputs/myInputs.module.css';
+
+const MyInputs = React.forwardRef((props, ref) => {
```

Comment:
dont write like this

write
import destructuring like {ForwardRef} from 'react